test/test_VanKeken.jl

```diff
@@ -1,19 +1,46 @@
 push!(LOAD_PATH, "..")
 using Test, Suppressor
 
-using ParallelStencil
-@init_parallel_stencil(Threads, Float64, 2)
+@static if ENV["JULIA_JUSTRELAX_BACKEND"] === "AMDGPU"
+    using AMDGPU
+    AMDGPU.allowscalar(true)
```

Comment:
`AMDGPU.allowscalar(true)` and `CUDA.allowscalar(true)` is only needed in `test_CellArrays2D.jl` and `test_CellArrays3D.jl`, you can drop it all the other tests.